#! /bin/sh
patch -p0 -f $* < $0
exit $?

2009-07-17  Marcus Brinkmann  <marcus@g10code.de>

	* gpgex.cc (start_help): Do not cripple the browser window.

Index: src/gpgex.cc
===================================================================
--- src/gpgex.cc	(revision 68)
+++ src/gpgex.cc	(working copy)
@@ -505,10 +505,10 @@
   /* FIXME: Pick a good configuration.  */
   // Only for IE7?
   // web->put_Resizable (VARIANT_TRUE);
-  web->put_ToolBar (FALSE);
-  web->put_AddressBar (VARIANT_FALSE);
-  web->put_MenuBar (VARIANT_FALSE);
-  web->put_StatusBar (VARIANT_FALSE);
+  // web->put_ToolBar (FALSE);
+  // web->put_AddressBar (VARIANT_FALSE);
+  // web->put_MenuBar (VARIANT_FALSE);
+  // web->put_StatusBar (VARIANT_FALSE);
   // width, height
   web->put_Visible (VARIANT_TRUE);
 
